providers/curation/github.js

```diff
@@ -109,13 +110,33 @@ class GitHubCurationService {
     // Github requires name/email to set committer
     if ((info.name || info.login) && info.email)
       fileBody.committer = { name: info.name || info.login, email: info.email }
+    else logger.info('Github requires name/email to set committer, info.email is currently: ', info.email)
     if (get(currentContent, '_origin.sha')) {
       fileBody.sha = currentContent._origin.sha
+      // Extract the component values
+      const { coordinates, revisions } = currentContent
+      const { name, provider, type } = coordinates
+      // Only get the latest revision
+      let newRevision = Object.keys(patch.revisions)[0];
+      let component = name + '/' + newRevision
+      try{
+        const validateResult = await this._validateComponentExists(component)
+      } catch (error) {
+        this.logger.info(`version not found: ${error.toString()}`)
+        return null;
```

Comment:
are we sure that we need to return a null value?

Reply:
resolved